clang/test/SemaSYCL/sycl-pfwg-invalid-code.cpp

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang_cc1 -fsycl-is-device %s -verify
+
+// Tests that an error diagnostic is issued without a crash (due to a
+// triggered assertion) when some invalid cases are encountered in
```

Comment:
Nit: This assertion no longer exists right? So I don't think this comment is valid anymore. I think you can just remove the content in parenthesis.

Reply:
No, the assertion remains at SemaSYCL.cpp:2517